app/vue3/src/client/preview/render.ts

```diff
@@ -14,7 +14,8 @@ export const render: ArgsStoryFn<VueFramework> = (props, context) => {
     );
   }
 
-  return h(Component, props);
+  // TODO remove this hack
+  return h(Component as Parameters<typeof h>[0], props);
```

Comment:
So multiple conflicting version of vue typings exits and they are equal but not the same, and so typescript complains. Since they are equal anyway, I simply cast it. this should result the same when users use it (because it will be the type they have installed, I assume